api/src/test/java/org/pmiops/workbench/access/UserServiceAccessTest.java

```diff
@@ -74,8 +74,7 @@
 
   private Function<Timestamp, Function<DbUser, DbUser>> registerUserWithTime =
       t -> dbu -> registerUser(t, dbu);
-  private Function<DbUser, DbUser> registerUserNow =
-      registerUserWithTime.apply(new Timestamp(PROVIDED_CLOCK.millis()));
```

Comment:
sometimes this would get set to a previous test's clock value somehow.

Moved this definition to ensure this can't happen.